Platon/backend/app/workspace_system/views.py

```diff
@@ -0,0 +1,154 @@
+from flask import make_response,jsonify,request
+from flask_restplus import Resource,Namespace, fields
+
+from app.auth_system.helpers import login_required
+from app.workspace_system.forms import TrendingProjectsForm,trending_project_parser,get_self_parser
+from app.workspace_system.models import Workspace,Contribution
+from app.auth_system.models import User
+from app.workspace_system.helpers import schedule_regularly
+
+
+from app import api, db
+
+workspace_system_ns = Namespace("Workspace System",
+                                description="Workspace System Endpoints",
+                                path = "/worksapce_system")
+
+contributor_model = api.model("Contributor", {
+    "id": fields.Integer,
+    "name": fields.String,
+    "surname": fields.String,
+    }
+)
+
+workspace_model = api.model('Worksapce', {
+    "id": fields.Integer,
+    "title": fields.String,
+    "is_private": fields.Integer,
+    "description": fields.String,
+    "state": fields.Integer,
+    "deadline": fields.String,
+    "creation_time": fields.String,
+    "max_contibutors": fields.Integer,
+    "contributors": fields.List(
+        fields.Nested(contributor_model)
+    ),
+})
+
+workspace_list_model = api.model('Worksapce List', {
+    'workspaces': fields.List(
+        fields.Nested(workspace_model)
+    )
+})
+
+workspace_small_model = api.model('Workspace', {
+    "id": fields.Integer,
+    "title": fields.String,
+    "description": fields.String,
+    "state": fields.Integer,
+    "contributors": fields.List(
+        fields.Nested(contributor_model)
+    ),
+})
+
+trending_workspaces_model = api.model('Trending Projects', {
+    'trending_projects': fields.List(
+        fields.Nested(workspace_small_model)
+    )
+})
+
+@workspace_system_ns.route("/self")
+class GetSelfWorkspaces(Resource):
```

Comment:
@HUmutOzdemir Actually, it might be better if we merge this code after "Read Workspace" functionaly gets merged; because there is a mismatch between the response formats of it and this endpoint here. After the merge, we should align their response formats.